frontend/public/index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="utf-8" />
   <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <meta name="description" content="Web site created using create-react-app" />
+  <meta name="description" content="논문간 인용관계 시각화" />
```

Comment:
```suggestion
  <meta name="description" content="논문 간 인용관계 시각화" />
```
`인용관계`도 `인용v관계`로 해야할 것 같지만,, 인용관계 를 하나의 단어로 보는 것도 좋을 것 같습니다.